airtable/params.py

```diff
@@ -1,368 +1,462 @@
+        return OrderedDict(sorted(param_dict.items()))
+
+
+class AirtableParams(object):
+    class MaxRecordsParam(_BaseParam):
+        """
+        Max Records Param
+
+        Kwargs:
+            ``max_records=`` or ``maxRecords=``
+
+        The maximum total number of records that will be returned.
+
+        Usage:
+
+        >>> airtable.get_all(max_records=10)
+
+        Args:
+            max_records (``int``): The maximum total number of records that
+                will be returned.
+
+
+        """
+
+        # Class Input > Output
+        # >>> filter = MaxRecordsParam(100)
+        # >>> filter.to_param_dict()
+        # {'maxRecords: 100}
+
+        param_name = "maxRecords"
+        kwarg = "max_records"
+
+    class ViewParam(_BaseParam):
+        """
+        View Param
+
+        Kwargs:
+            ``view=``
+
+        If set, only the records in that view will be returned.
+        The records will be sorted according to the order of the view.
+
+        Usage:
+
+        >>> airtable.get_all(view='My View')
+
+        Args:
+            view (``str``): The name or ID of a view.
+
+        """
+
+        # Class Input > Output
+        # >>> filter = ViewParam('Name or Id Of View')
+        # >>> filter.to_param_dict()
+        # {'view: 'Name or Id Of View'}
+
+        param_name = "view"
+        kwarg = param_name
+
+    class PageSizeParam(_BaseParam):
+        """
+        Page Size Param
+
+        Kwargs:
+            ``page_size=`` or ``pageSize=``
+
+        Limits the maximum number of records returned in each request.
+        Default is 100.
+
+        Usage:
+
+        >>> airtable.get_all(page_size=50)
+
+        Args:
+            page_size (``int``): The number of records returned in each request.
+                Must be less than or equal to 100. Default is 100.
+
+        """
+
+        # Class Input > Output
+        # >>> filter = PageSizeParam(50)
+        # >>> filter.to_param_dict()
+        # {'pageSize: 50}
+
+        param_name = "pageSize"
+        kwarg = "page_size"
+
+    class CellFormatParam(_BaseParam):
+        """
+        cellFormat Param
+
+        Kwargs:
+            ``cellFormat=``
+
+        If set, the format that should be used for cell values. Supported values are:
+           json: cells will be formatted as JSON, depending on the field type.
+           string: cells will be formatted as user-facing strings, regardless of the field type.
+                   Note: You should not rely on the format of these strings, as it is subject to change.
+
+        The default is json.
+
+        Usage:
+
+        >>> airtable.get_all(cellFormat='json')
+
+        Args:
+            cellFormat (``str``): The format to use.
+
+        """
+
+        # Class Input > Output
+        # >>> filter = CellFormatParam('The format to use.')
+        # >>> filter.to_param_dict()
+        # {'cellFormat: 'The format to use.'}
+
+        param_name = "cellFormat"
+        kwarg = param_name
```

Comment:
`kwarg` can be used to allow for standard python formatting eg (snake case) or more concise param. name. For example, here we could use `kwarg = cell_format` or event `kwarg = format`
These were implemented so user can always use the official airtable param names, but can also use custom library "aliases" for cleaner python-styled kwarg names

eg. `param_name = filterByFormula=` -> `kwarg= formula`